Spec/RestClient.swift

```diff
@@ -565,6 +565,72 @@ class RestClient: QuickSpec {
             // RSC15
             context("Host Fallback") {
 
+                // TO3k7
+                context("fallbackHostsUseDefault option") {
+
+                    it("allows the default fallback hosts to be used when @environment@ is not production") {
+                        let options = ARTClientOptions(key: "xxxx:xxxx")
+                        options.environment = "not-production"
+                        options.fallbackHosts = ["fakeA.ably.io", "fakeB.ably.io"]
+                        options.fallbackHostsUseDefault = true
+
+                        let client = ARTRest(options: options)
+                        expect(client.options.fallbackHostsUseDefault).to(beTrue())
+                        // Not production
+                        expect(client.options.environment).toNot(beNil())
+                        expect(client.options.environment).toNot(equal("production"))
+
+                        let fallback = ARTFallback(options: client.options)
+                        expect(fallback.hosts).to(haveCount(ARTDefault.fallbackHosts().count))
+
+                        ARTDefault.fallbackHosts().forEach() {
+                            expect(fallback.hosts).to(contain($0))
+                        }
+                    }
+
+                    it("allows the default fallback hosts to be used when a custom Realtime or REST host endpoint is being used") {
+                        let options = ARTClientOptions(key: "xxxx:xxxx")
+                        options.restHost = "fake1.ably.io"
+                        options.realtimeHost = "fake2.ably.io"
+                        options.fallbackHosts = ["fakeA.ably.io", "fakeB.ably.io"]
+                        options.fallbackHostsUseDefault = true
```

Comment:
As above, this says "allows the default fallback hosts to be used" yet you're providing fallback hosts in the Array